gsplat/cuda/csrc/forward.cu

```diff
@@ -277,7 +277,11 @@ void bin_and_sort_gaussians(
 
     // sort intersections by ascending tile ID and depth with RadixSort
     int32_t max_tile_id = (int32_t)(tile_bounds.x * tile_bounds.y);
+#if defined(WIN32) || defined(_WIN32) || defined(__WIN32) && !defined(__CYGWIN__)
```

Comment:
thanks for dealing with the different os builtins, I didn't think of this

Reply:
I think this is from some other PR actually... I'm not sure why its showing up here